index.js

```diff
@@ -8,43 +8,65 @@ var hasFlag = function (flag) {
 	return pos !== -1 && (terminator !== -1 ? pos < terminator : true);
 };
 
-module.exports = (function () {
-	if ('FORCE_COLOR' in process.env) {
-		return true;
+var support = function (level) {
+	if (level <= 0) {
+		return null;
```

Comment:
No I understand that. I think I see what I mean; I was returning `null` in order to keep things backwards compatible. I can remove that functionality and we can always return an object; it'd just be a major bump.